### PR TITLE
Added parsers

### PR DESCRIPTION
**Pull request by iulianpascalau**
- added parsers

### CODE REVIEW

File: parsers/listParser.go [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+	}
+
+	parser.mut.Lock()
+	defer parser.mut.Unlock()
+
+	parser.emptyLists()
+
+	spitLines := strings.Split(string(dataFile), "\n")
+	for index, line := range spitLines {
+		line = strings.Trim(line, " \t\r\n")
```
Comment (andreibancioiu) [inline parsers/listParser.go:46]: Does `strings.TrimSpace` help in this context?

**Review by andreibancioiu — APPROVED** [review]

**Review by miiu96 — APPROVED** [review]

File: parsers/listParser.go [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+	"github.com/multiversx/mx-chain-core-go/core/pubkeyConverter"
+	"github.com/multiversx/mx-chain-keys-monitor-go/core"
+)
+
+const commentMarker = "#"
+
+var bech32PubKeyConverter, _ = pubkeyConverter.NewBech32PubkeyConverter(core.AddressLen, core.AddressHRP)
+
+type listParser struct {
+	filename   string
```
[the remark this answers is not in the document]
Comment (iulianpascalau) [reply parsers/listParser.go:19]: will do this in the next PR

File: parsers/listParser.go [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+
+// NewListParser creates a new file list parser
+func NewListParser(filename string) *listParser {
+	return &listParser{
+		filename: filename,
+	}
+}
+
+// ParseFile will try to parse to file and split the identities in 2. Errors if something is wrong with the file
+func (parser *listParser) ParseFile() error {
```
[the remark this answers is not in the document]
Comment (iulianpascalau) [reply parsers/listParser.go:33]: will do this in the next PR

File: parsers/listParser.go (hunk 1 above)
Comment (iulianpascalau) [reply parsers/listParser.go:46]: I guess, will do this in the next PR

Comment (iulianpascalau) [reply parsers/listParser.go:46]: yup, works, will be done in the next PR

**Pull request merged**